UnityProject/Assets/Scripts/Objects/Cargo/ConveyorBelts/ConveyorBelt.cs

```diff
@@ -42,6 +42,12 @@ public class ConveyorBelt : NetworkBehaviour, ICheckedInteractable<HandApply>, I
 		private void Awake()
 		{
 			registerTile = GetComponent<RegisterTile>();
+			if(netIdentity.isServer) RefreshSprites(); //So the host can see the sprites animated correctly on start.
```

Comment:
`netIdentity.isServer` pretty sure isnt a way to check if its a server doing the action. 

You shouldnt need to do the awake check now that OnSpawnServer is used

Reply:
OnSeverSpawn() doesn't work on the host